cython_lint/cython_lint.py

```diff
@@ -17,6 +19,11 @@
 from typing import NoReturn
 from typing import Sequence
 
+if sys.version_info >= (3, 11):  # pragma: no cover
+    import tomllib
+else:
+    import tomli as tomllib
```

Comment:
doesn't this need to be a runtime dependency now?

Reply:
I guess so. I added it to the ``install_requires section``